src/tools/tidy/src/tests_revision_unpaired_stdout_stderr.rs

```diff
@@ -58,7 +58,7 @@ pub fn check(tests_path: impl AsRef<Path>, bad: &mut bool) {
 
             let mut expected_revisions = BTreeSet::new();
 
-            let contents = std::fs::read_to_string(test).unwrap();
+            let Ok(contents) = std::fs::read_to_string(test) else { continue };
```

Comment:
...this, and another line in `md-doc` blow up if a *test* has non-utf-8 bytes.

I ended up removing a test of "`.rs` file with invalid few utf-8 bytes" because md-docs is in a separate repo. I feel we should make the entire test suite more resilient to these, but I think I can make that test I added and removed be in `run-make`...